src/scrapers/tools/geocoder.py

```diff
@@ -62,58 +72,86 @@ def geocode(self, item, feature_type=None):
         # item.location prepareren voor stadsdelen
         geocode_term = geocode_term.replace("Amsterdam Algemeen", "Amsterdam")
 
-        # is the item.location National?
+        matchers = [self.match_nederland, self.match_stadsdeel, self.match_nominatim, self.match_nominatim_nlve_muni]
+        match = None
+        for matcher in matchers:
+            match = matcher(item, geocode_term, feature_type, id_array)
+            if match:
+                break
+
+        if not match:
+            match = Match(geocode_term, None, None)
+            logger.warning(geocode_term + " not found, " + str(len(id_array)) +
+                                                   " entries set to original`" +
+                                                   geocode_term + "`")
+
+        self.bulk_update(match.address, match.lat, match.lon, id_array)
+
+        time.sleep(1)  # Sleep so we don"t overstretch the nominatim api
```

Comment:
Zou je de sleep niet willen doen daar waar er wordt geitereerd ipv in de uitvoer-method?

Reply:
Niet iedere geocode strategy gaat langs nominatim momenteel. Dus dat zou overbodige sleeps opleveren dan.